src/staticanalysis/bot/static_analysis_bot/report/base.py

```diff
@@ -150,6 +154,12 @@ def pluralize(word, nb):
         if analyzers:
             comment += COMMENT_RUN_ANALYZERS.format(analyzers='\n'.join(analyzers))
 
+        if extra_issues:
+            comment += COMMENT_EXTRA_ISSUES.format(issues='\n'.join([
+                '* {} on line {} : {}'.format(issue.path, issue.line, issue.as_text())
+                for issue in extra_issues
```

Comment:
Should we limit them in number?

Reply:
I don't know if we'll have a lot... we are a bit in the dark here